### ISSUE TITLE
bcgTree directory permissions issue

### ISSUE THREAD

**Issue by alvarosimon** [easybuilders/easybuild-easyconfigs#6123]
This was detected in our clusters regarding bcgTree module installation:
```
module load bcgTree/1.0.10-intel-2018a-Perl-5.26.1
$ bcgTree.pl --check-external-programs
Gblocks   !fail!    
RAxML         ok    /apps/gent/CO7/haswell-ib/software/RAxML/8.2.11-intel-2018a-hybrid-avx2/bin/raxmlHPC
hmmsearch     ok    /apps/gent/CO7/haswell-ib/software/HMMER/3.1b2-intel-2018a/bin/hmmsearch
muscle        ok    /apps/gent/CO7/haswell-ib/software/MUSCLE/3.8.31-intel-2018a/bin/muscle
```

Gblocks directory is only accessible by the installation user:

```
$ ls /apps/gent/CO7/haswell-ib/software/Gblocks/0.91b/Gblocks
ls: cannot access /apps/gent/CO7/haswell-ib/software/Gblocks/0.91b/Gblocks: Permission denied
```

**Comment by alvarosimon** [comment]
Fixed manually for the moment:

```
# chmod +x /apps/gent/CO7/haswell-ib/software/Gblocks/0.91b/more_alignments
# chmod +x /apps/gent/CO7/haswell-ib/software/Gblocks/0.91b
# chmod +x /apps/gent/CO7/sandybridge/software/Gblocks/0.91b/more_alignments
# chmod +x /apps/gent/CO7/sandybridge/software/Gblocks/0.91b
```

**Comment by boegel** [comment]
fixed in cc727405e which is included in #6057
